### ISSUE TITLE
Shapeshift not removing movement-impairing effects

### ISSUE THREAD

**Issue by roblockster** [dalaranwow/dalaran-wow#4815]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Druid shapeshifting is not removing movement-impairing debuffs on Yogg Saron encounter (and likely many other such debuffs)

**Expected behaviour**: "The act of shapeshifting frees the caster of... Movement Impairing effects."

**Steps to reproduce the problem**:

1. Be a druid
2. Get a movement-impairing debuff from Yogg Saron corrupter tentacle
3. Shapeshift in and out, powershift, etc. while RP-walking around the room since the movement-impairing debuffs are not removed.

**Include proofs for this behaviour** 

It's in the tooltip of the ability. 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by FireheartBDG** [comment]
I watched two videos of ferals playing yogg saron back in wrath, neither shape-shifted Apathy off. It's likely it cannot be shifted and HAS to be dispelled, as shape shift historically has very few exceptions to what cannot be shifted off (JoJ from paladins for example)

**Comment by bestcrit** [comment]
Shapeshift doesn't remove acidmaw's poison either, for sure. I think it should do that.

**Comment by FireheartBDG** [comment]
@bestcrit It isn't supposed to remove some boss mechanics, the acidmaw debuff HAS to be removed by burning bile
